features/admin.identity-providers.v1/components/forms/authenticators/push-authenticator-form.tsx

```diff
@@ -173,6 +173,7 @@ export const PushAuthenticatorForm: FunctionComponent<PushAuthenticatorFormProps
     // This can be used when `meta` support is there.
     const [ , setFormFields ] = useState<PushAuthenticatorFormFieldsInterface>(undefined);
     const [ initialValues, setInitialValues ] = useState<PushAuthenticatorFormInitialValuesInterface>(undefined);
+    const [ isReadOnly ] = useState<boolean>(isSubOrganization() || readOnly);
```

Comment:
Your code is an anti-pattern because useState initializes once and does not update when isSubOrganization() or readOnly change. Instead, compute the value directly inside render:

```
const isReadOnly = isSubOrganization() || readOnly;
```